src/main/java/team/chisel/block/BlockCarvableNeonite.java

```diff
@@ -0,0 +1,19 @@
+package team.chisel.block;
+
+import net.minecraft.block.material.Material;
+import net.minecraft.entity.EnumCreatureType;
+import net.minecraft.world.IBlockAccess;
+
+import com.cricketcraft.chisel.api.ICarvable;
+
+public class BlockCarvableNeonite extends BlockCarvable implements ICarvable {
```

Comment:
No need to implement ICarvable as BlockCarvable already does so and this class inhertits from it.

Also, this class and `BlockCarvableGlotek` do exactly the same, consider merging them into e.g. `BlockCarvableNonSpawnable`.